### PR TITLE
added order controllers

### PR DESCRIPTION
**Pull request by Martynice**

### CODE REVIEW

File: src/main/java/com/internet/shop/controllers/order/CompleteOrderController.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    private static final Injector injector = Injector.getInstance("com.internet.shop");
+    private final OrderService orderService =
+            (OrderService) injector.getInstance(OrderService.class);
+    private final ShoppingCartService shoppingCartService =
+            (ShoppingCartService) injector.getInstance(ShoppingCartService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws IOException {
+        orderService.completeOrder(shoppingCartService.getByUserId(USER_ID));
```
Comment (andromash) [inline src/main/java/com/internet/shop/controllers/order/CompleteOrderController.java:24]: According to the checklist, we should separate calling via service into the local variable. still, it has no impact on the work :)

**Review by andromash — APPROVED** [review]
Good job!

File: src/main/java/com/internet/shop/controllers/order/CompleteOrderController.java (hunk 1 above)
Comment (RomanDubovskyi) [reply src/main/java/com/internet/shop/controllers/order/CompleteOrderController.java:24]: That point is concerning situation when you call service method that fetcher data from db, for example  shoppingCartService.getByUserId() several times. It's better to save result into local variable in such case.

**Review by RomanDubovskyi — APPROVED** [review]
looks good to me

**Pull request merged**